src/index.js

```diff
@@ -280,35 +282,37 @@ export const setLinkElementData = ($el, manifest) => {
   }
 };
 
-export const initialiseEmbed = (mediaMode) => {
+export const initialiseEmbed = (mediaMode, language) => {
 
   playerWrapper.removeClass('loading');
 
   let manifestJsonld = player.manifest.__jsonld;
 
-  initialiseAttribution(manifestJsonld, mediaMode);
-  setLinkElementData($('.title-link'), manifestJsonld);
+  initialiseAttribution(manifestJsonld, mediaMode, language);
+  setLinkElementData($('.title-link'), manifestJsonld, language);
+
   $('.logo-link').removeAttr('style');
 
   if (duration === -1 && manifestJsonld.items[0].duration) {
     duration = manifestJsonld.items[0].duration;
   }
 };
 
-export const initialisePlayer = (mediaUrl, mediaMode) => {
-  let p = new EuropeanaMediaPlayer(playerWrapper, { manifest: mediaUrl }, { mode: 'player', manifest: mediaUrl });
+export const initialisePlayer = (mediaUrl, mediaMode, language) => {
+  let p = new EuropeanaMediaPlayer(playerWrapper, { manifest: mediaUrl }, { mode: 'player', manifest: mediaUrl, language });
```

Comment:
Are you passing here only the language from the url parameter? The option should be set as `language: language` (see https://github.com/europeana/media-player#editor-and-language-example)

Reply:
That fails linting, with the recommendation to just pass it as "language" (ES6 syntax)

Reply:
Ok, then I have no further comments